aroma/tests/test_features.py

```diff
@@ -8,43 +9,35 @@
 from aroma.tests.utils import get_tests_resource_path
 
 
-def test_feature_time_series(nilearn_data):
+def test_feature_time_series(mel_mix, mc, features_df):
 
-    # Get path to confounds file
-    test_path, _ = op.split(nilearn_data.func[0])
+    # Run feature_time_series
+    max_RP_corr = features.feature_time_series(mel_mix, mc)
 
-    # Read confounds
-    confounds = pd.read_csv(nilearn_data.confounds[0], sep="\t")
+    # Read features csv
+    df = pd.read_csv(features_df)
 
-    # Extract motion parameters from confounds
-    mc = confounds[["rot_x", "rot_y", "rot_z", "trans_x", "trans_y", "trans_z"]]
-    mc_path = op.join(test_path, "mc.tsv")
-    mc.to_csv(mc_path, sep="\t", index=False, header=None)
+    assert np.allclose(df["max_RP_corr"].values[:5], max_RP_corr[:5], atol=1e-1)
```

Comment:
Given that the IC time series are not always exactly the same, it is tricky to come up with a robust assert here.

Can you folks think of a better way of asserting this?

Reply:
the key is what exactly you want to test - for a unit test you might want to fixate some IC time series so that you always get the exact same result.

Reply:
Now that the ICA time series are kept constant, I think you can test across all columns, right?